docs/source/getting_started/installation.rst

```diff
@@ -15,6 +15,33 @@ JupyterLab can be installed using ``conda``, ``mamba``, ``pip``, ``pipenv`` or `
     upgrade procedures to account for possible breaking changes that may disrupt your usage
     of JupyterLab and any related tools that are critical to your workflows.
 
+
+JupyterLab Desktop
+------------------
+JupyterLab Desktop is the easiest way to get started with Jupyter. Use the proper download link below to install for your platform
+    .. raw:: html
+
+	 <div id="downloads" class="unknown">
+	 <h2> Download JupyterLab </h2>
+	 <a class="osLink MacIntel" href="/osx.dmg">Download for OS X</a>
+	 <a class="osLink Linux" href="linux.snap">Download for linux</a>
+	 <a class="osLink Win32" href="windows.exe">Download for Windows</a>
```

Comment:
And here is the main problem how to get the proper links. I would suggest to implement a GitHub pages website in JupyterLab desktop that have dummy pages on fix URLs that redirect automatically to the proper download links.